spec/controllers/people_controller_spec.rb

```diff
@@ -20,139 +20,30 @@
 
 RSpec.describe PeopleController, type: :controller do
 
-  # This should return the minimal set of attributes required to create a valid
-  # Person. As you add validations to Person, be sure to
-  # adjust the attributes here as well.
-  let(:valid_attributes) {
-    skip("Add a hash of attributes valid for your model")
-  }
+  describe "GET show.json" do
+    let!(:person) { create(:person) }
+    subject(:show_person) { get :show, params: { id: person.id, format: :json } }
 
-  let(:invalid_attributes) {
-    skip("Add a hash of attributes invalid for your model")
-  }
+    it "assign @person " do
```

Comment:
It is best for these specs to be in third person, for example: `it "assigns @person"`